Insight.Database/Reliable/ReliableCommand.cs

```diff
@@ -47,6 +47,7 @@ public override int ExecuteNonQuery()
 				() =>
 				{
 					FixupParameters();
+                    InnerConnection.AutoOpen();

```

Comment:
Once we rename to EnsureOpened or EnsureIsOpen, then this makes sense.
When you use Insight, we should never hit this section with a closed connection.